nss-day-cohort-34/bangazon-site-the-naughty-list
# Skf back button and my products

# Description
"My Products" view now includes the number of products sold. A "Go Back" link was added to Product Details that dynamically routes the user to the previous page.

Related to Issue #20  


# Testing Instructions
- Click on "My Products"
- Confirm that you can see both the quantity of each product and the number sold
- Click on a "Details" link next to one of your products
- Click the "Go Back" link and confirm that it returns you to the "My Products" view
- Go to the Homepage then click on a product link to enter the Product Details view
- Confirm that it returns you to the Homepage

# Checklist:
